idd/Energy+.idd.in

```diff
@@ -34730,6 +34730,4646 @@ ZoneHVAC:EvaporativeCoolerUnit,
+        \units minutes
+        \default 10
+   A13, \field First Fuel Type
+        \type choice
+        \key None
+        \key Electricity
+        \key NaturalGas
+        \key PropaneGas
+        \key FuelOil#1
+        \key FuelOil#2
+        \key Diesel
+        \key Gasoline
+        \key Coal
+        \key OtherFuel1
+        \key OtherFuel2
+        \key Steam
+        \key DistrictHeating
+        \key DistrictCooling
+		\note Select the fuel type associated with field: "System Electric Power Lookup Table" in each mode.
+        \note If this field is blank, default first fuel type = Electricity.
+        \default Electricity 
+   A14, \field Second Fuel Type
+        \type choice
+        \key None
+        \key Electricity
+        \key NaturalGas
+        \key PropaneGas
+        \key FuelOil#1
+        \key FuelOil#2
+        \key Diesel
+        \key Gasoline
+        \key Coal
+        \key OtherFuel1
+        \key OtherFuel2
+        \key Steam
+        \key DistrictHeating
+        \key DistrictCooling
+        \note Select the fuel type associated with field: "System Second Fuel Consumption Lookup Table" in each mode.
+        \note If this field is blank, default second fuel type = None.
+        \default None
+   A15, \field Third Fuel Type
+        \type choice
+        \key None
+        \key Electricity
+        \key NaturalGas
+        \key PropaneGas
+        \key FuelOil#1
+        \key FuelOil#2
+        \key Diesel
+        \key Gasoline
+        \key Coal
+        \key OtherFuel1
+        \key OtherFuel2
+        \key Steam
+        \key DistrictHeating
+        \key DistrictCooling
+        \note Select the fuel type associated with field: "System Third Fuel Consumption Lookup Table" in each mode.
+        \note If this field is blank, default third fuel type = None.
+        \default None
+   A16, \field Objective Function to Minimize
+        \type choice
+        \key Electricity Use
+        \key Second Fuel Use
+        \key Third Fuel Use
+        \key Water Use
+        \note In each time step, controlled variables will be chosen to minimize the selection in this field, subject to constraints.
+        \note If this field is blank, the objective function will minimize electricity use.
+        \default Electricity Use
+   A17, \field Design Specification Outdoor Air Object Name
+        \type object-list
+        \object-list DesignSpecificationOutdoorAirNames
+        \note Enter the name of a DesignSpecification:OutdoorAir object. Information in that object will be used to compute the minimum outside air flow rate in each time step.
+        \note If this field is blank, the system may still supply outside air, if it is capable as described by lookup tables, when doing so is the most efficient way to satisfy other constraints.
+   A18, \field Mode 0 Name
+        \type alpha
+        \retaincase
```

Comment:
Eventually, but I wouldn't worry about right now. EnergyPlus code is heavily biased towards uppercase strings that it'll be on a case by case basis as new objects are added and refactored.